src/platform/silabs/rs911x/rsi_ble_config.h

```diff
@@ -24,6 +24,11 @@
  * *                      Macros
  * ******************************************************/
 //! application event list
+#ifndef RSI_FAILURE
+// failure return value
+#define RSI_FAILURE -1
+#endif

```

Comment:
why is the `ifndef` necessary? seems odd that have this line?

Reply:
Can you add a comment with a TODO to remove these lines when we transition to the new SDK?